Sources/NIOPosix/SelectableEventLoop.swift

```diff
@@ -299,6 +299,19 @@ Further information:
         }, .now()))
     }
 
+    #if swift(>=5.9)
+    @available(macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0, *)
+    @usableFromInline
+    func enqueue(_ job: consuming ExecutorJob) {
+        let scheduledTask = ScheduledTask(
+            id: self.scheduledTaskCounter.loadThenWrappingIncrement(ordering: .relaxed),
+            job: job,
+            readyTime: .now()
+        )
+        try? self._schedule0(scheduledTask)
```

Comment:
Can you reproduce the comment above to express why we don't catch an error here.